src/components/FeaturedProject.css

```diff
@@ -66,14 +66,15 @@
   text-decoration: none;
 }
 
-.project__button:hover{
-  box-shadow: 2px 4px #4dd6a8;
-  /*(--turquoise-green);*/
-  transform: translateY(-0.2em);
-}
+@media (min-width: 400px){
+  .project__button:hover{
+    box-shadow: 2px 4px #4dd6a8;
```

Comment:
Could you replace the box-shadow hex color with the global variable `--primary-green` provided in App.css?